organizations/admin.py

```diff
@@ -12,8 +12,8 @@ class OrganizationAdmin(admin.ModelAdmin):
     Admin for the Organization table.
     soft-delete on the organizations
     """
-    list_display = ('name', 'description', 'created', 'active')
-    fields = ('name', 'description', 'created', 'active')
+    list_display = ('name', 'description', 'logo', 'created', 'active')
+    fields = ('name', 'description', 'logo', 'created', 'active')
```

Comment:
I don't think we need `fields` declaration here unless we are limiting the number editable fields on admin site.